src/osx/cocoa/nonownedwnd.mm

```diff
@@ -335,6 +337,27 @@ - (id) init
     return self;
 }
 
+- (NSApplicationPresentationOptions)window:(NSWindow *)window
+      willUseFullScreenPresentationOptions:(NSApplicationPresentationOptions)proposedOptions
```

Comment:
Looking at [the docs](https://developer.apple.com/documentation/appkit/nswindowdelegate/1419144-window?language=objc) it seems like we are supposed to use `proposedOptions` by default, so I wonder if it's correct to just ignore them?

No other comments from me and if you think this is fine and if Stefan doesn't say anything neither, I'll merge this soon.

Thanks again!

Reply:
I think it's OK not to use the `proposedOptions` as we don't implement presentation options in the application delegate (which would be the source for the `proposedOptions` value if I'm not mistaken).